tasks/post-deploy/04-add-default-preconnect-domains.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Set domains for the Preconnect Settings option
+echo -e "https://in.hotjar.com\nhttps://act.greenpeace.org" > domains.txt
+cat domains.txt | wp option patch update planet4_options preconnect_domains
```

Comment:
Can you replace that with `wp option patch insert planet4_options preconnect_domains < domains.txt` to make shellcheck happy?

Notice I used `insert` instead of `update`, since that option key won't be there to update.

Reply:
Updated.